libr/bin/dwarf.c

```diff
@@ -456,8 +469,9 @@ static const ut8* r_bin_dwarf_parse_ext_opcode(const RBin *a, const ut8 *obuf,
 
 	opcode = *buf++;
 
+	// Maybe add offset to the print later?
```

Comment:
Add TODO or FIXME in these cases, so it will be shown in the respective searches or plugins